test/unit/features/directives/on.spec.js

```diff
@@ -102,6 +102,24 @@ describe('Directive v-on', () => {
     expect(callOrder.toString()).toBe('1,2')
   })
 
+  it('should support once', () => {
+    const numberPushed = []
+    vm = new Vue({
+      el,
+      template: `
+        <div @click.once="foo">
+        </div>
+      `,
+      methods: {
+        foo () { numberPushed.push(1) }
```

Comment:
Because the second click will be triggered very fast in the unit test, the spy will be called twice even if the handler is removed in the first call. Could you please tell me how to hold the test for a while between the two clicks.

Will there be any side effect if I use setTimeout for the second click?